### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by N0xMare**
https://github.com/valory-xyz/ai-registry-mech-frontend/assets/22061815/19344604-68c5-4bf4-a2f6-19e1ddb15da1

### CODE REVIEW

File: components/Docs/index.jsx [hunk 1]
```diff
@@ -45,10 +45,11 @@ Request
 ~~~json
 {
     "prompt": "The request prompt goes here",
-    "tool": "The tool we want the off-chain agent to use goes here, for example openai-gpt4",
+    "tools": ["The tools we want to use go here", "can be multiple or just one"],
     "nonce": "The uuid goes here"
 }
 ~~~
+If the request specifies tools that the Agent has specified in its metadata (also on IPFS) then the request is valid and the Agent will perform the task. If the request uses tools that the Agent has not specified in its metadata then the request is invalid and the Agent will not perform the task.
```
Comment (DavidMinarsch) [inline components/Docs/index.jsx:52]: Please provide a sample of the agent metadata file. Use info from here too: https://aimechs.autonolas.network/

Comment (DavidMinarsch) [reply components/Docs/index.jsx:52]: See also the metadata generator on the factory. It should all link up. I think we're still missing the tool list there

Comment (DavidMinarsch) [reply components/Docs/index.jsx:52]: Same applies to that form - coordinate with @mohandast52 to get the changes implemented

**Comment by N0xMare** [comment]
@mohandast52 I added in the agent metadata. The other change you can see above is a change to the data format on IPFS to make the request more intuitive as it takes a list of strings (tools) etc..Wherever data is being is stored to IPFS for the requests from the front-end should be changed to reflect "tools" instead of "tool" as is currently is implemented in main.

**Pull request closed without merging**